apps/web/src/views/Notifications/containers/OnBoardingView.tsx

```diff
@@ -24,6 +24,8 @@ interface IOnBoardingProps {
   identityKey: string | undefined
   handleRegistration: () => Promise<void>
   isReady: boolean
+  isSubscribing: boolean
+  setIsSubscribing: Dispatch<SetStateAction<boolean>>
```

Comment:
it means `onSubscribe`? 
```suggestion
  onSubscribe: () => void
```